test/conftest.py

```diff
@@ -310,13 +310,6 @@ def pudl_sql_io_manager(
                         "config": pudl_datastore_config,
                     },
                 },
-                "ops": {
-                    "hourly_emissions_epacems": {
-                        "config": {
-                            "partition": True,
-                        }
-                    }
-                },
```

Comment:
I removed this config since it seemed to be breaking the integration tests and it seemed analogous to the changes you'd pushed for the CLI tools, but I wasn't 100% sure this was the right thing to do.